wbooks/src/redux/book/reducers.ts

```diff
@@ -1,34 +1,19 @@
-import { BookActions, BookState, BookTypes } from '@interfaces/book';
+import { completeReducer, completeState, createReducer } from 'redux-recompose';
+import { BookState } from '@interfaces/book';
 
-const initialState: BookState = {
-  booksLoading: false,
-  books: []
-};
-
-function bookReducer(state: BookState = initialState, action: BookActions): BookState {
-  switch (action.type) {
-    case BookTypes.BOOK_LIST_REQUEST:
-      return {
-        ...state,
-        booksLoading: true
-      };
+import { actions } from './actions';
 
-    case BookTypes.BOOK_LIST_REQUEST_SUCCESS:
-      return {
-        ...state,
-        books: action.payload,
-        booksLoading: false
-      };
+export const initialState: BookState = completeState({
+  description: {
+    booksLoading: false,
+    books: []
+  }
+});
```

Comment:
I think this should look like this:
```suggestion
export const initialState: BookState = completeState({
    booksLoading: false,
    books: []
});
```